package.json

```diff
@@ -33,6 +33,7 @@
         "cytoscape-grid-guide": "^2.3.0",
         "diff-match-patch": "^1.0.4",
         "echarts": "^5.5.1",
+        "eslint-plugin-no-only-tests": "^3.1.0",
```

Comment:
This is a dev dependency.
Not sure we should use this one at all. Having a library that adds only a single lint rule seems like peak JavaScript madness. We are better off adding the Mocha ESLint plugin to get a rule for blocking `only` tests along with other rules that also apply to Cypress. This is also what the Cypress eslint GitHub recommends for this check.
[
eslint-plugin-no-only-tests](https://github.com/cypress-io/eslint-plugin-cypress?tab=readme-ov-file#mocha-only-and-skip)